versions.txt

```diff
@@ -2,7 +2,7 @@
 # by default with the OpenTelemetry Operator. This would usually be the latest
 # stable OpenTelemetry version. When you update this file, make sure to update the
 # the docs as well.
-opentelemetry-collector=0.85.0
+opentelemetry-collector=0.86.0
```

Comment:
I'm not sure I can, since v0.86.0 is the first version to include the debug exporter. if it's preferable, i'm ok waiting for the new operator to release before merging this PR

Reply:
It's fine to keep it here, the collector version can be bumped before the release (e.g. to implement upgrade procedures)

Reply:
Though, not sure if our upgrade test can handle that, The job failed I have restarted it

Reply:
new release is in, so this should be okay. @codeboten can you rebase on main?